### PR TITLE
Enable Dependabot security scanning

### PR DESCRIPTION
**Pull request by svenaas**
Part of #458 

## Changes proposed in this pull request:
- Adds `.github/dependabot.yml` configuration file to enable suitable security scanning

## security considerations
None. This adds a security scanning mechanism.

### CODE REVIEW

**Comment by svenaas** [comment]
Adapted from `pages-core` per @drewbo's [suggestion](https://github.com/cloud-gov/pages-build-container/issues/458#issuecomment-2103199650)

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: pip
+    directory: /
+    schedule:
+      interval: weekly
+    # Disable version updates for npm dependencies
```
Comment (drewbo) [inline .github/dependabot.yml:7]:
```suggestion
    # Disable version updates for pip dependencies
```

**Review by drewbo — DISMISSED** [review]
I'm not positive this will work (because the other one isn't totally working) but fine to test for now

File: .github/dependabot.yml (hunk 1 above)
Comment (svenaas) [reply .github/dependabot.yml:7]: Whoops. Thanks!

**Review by drewbo — APPROVED** [review]

**Pull request merged**